docs/en/ingest-management/commands.asciidoc

```diff
@@ -282,6 +284,14 @@ Service token to use for communication with {es}.
 Force overwrite of current configuration without prompting for confirmation.
 This flag is helpful when using automation software or scripted deployments.
 
+Note: if the {agent} is already installed on the host, using --force may result in
+unpredictable behaviour with duplicate {agent}s appearing in {fleet}
```

Comment:
```suggestion
+
NOTE: If the {agent} is already installed on the host, using `--force` may result in
unpredictable behaviour with duplicate {agent}s appearing in {fleet}.
```

Reply:
`NOTE` is required if you want this to render using the note formatting. Also added some markup for the flag name and a period at the end of the sentence.